CMakeLists.txt

```diff
@@ -104,8 +104,8 @@ endif(${WITH_AIO})
 
 option(WITH_FUSE "Fuse is here" ON)
 if(${WITH_FUSE})
-find_package(fuse REQUIRED)
-set(HAVE_LIBFUSE ${FUSE_FOUND})
+find_package(fuse)
```

Comment:
i think we should fail at the cmake phase if the system has not have the necessary library/header files/program installed, even an option is ON by default. otherwise the user could pass `-DWITH_FUSE=ON` to cmake but end up without fuse. also this behaviour matches that of `configure`.